cmd/clusterawsadm/cloudformation/bootstrap/iam.go

```diff
@@ -47,13 +48,13 @@ func (t Template) GenerateManagedIAMPolicyDocuments(policyDocDir string) error {
 
 		pds, err := converters.IAMPolicyDocumentToJSON(*pd)
 		if err != nil {
-			return fmt.Errorf("failed to marshal policy document for ManagedIAMPolicy %q: %v", pn, err)
+			return errors.Wrapf(err, "failed to marshal policy document for ManagedIAMPolicy %q", pn)
```

Comment:
Should this use go v1.13 style wrapping rather than pkg/errors?
```suggestion
			return fmt.Errorf("failed to marshal policy document for ManagedIAMPolicy %q: %w", pn, err)
```